src/appmixer/microsoft/sharepoint/FindFilesOrFolders/FindFilesOrFolders.js

```
@@ -0,0 +1,75 @@
+    async receive(context) {
+        const { driveId, parentPath, fileTypesRestriction, outputType } = context.messages.in.content;
+        let { q } = context.messages.in.content;
+
+        if (!q) q = '';
+        const encodedQuery = encodeURIComponent(q);
+
+        let filesAndFolder = [];
+
+        let url;
+
+        // Different URL is only used, because using /root:/<parentPath>:/search is unreliable, for example, it doesn't return all images even when searched for them using the query
```

ESLint

[error] This line has a comment length of 181. Maximum allowed is 150.

(max-len)